tests/net_inet/asyncio_tls_open_connection.py

```diff
@@ -0,0 +1,51 @@
+# Test simple HTTPS request with uasyncio.open_connection()
```

Comment:
I don't think this test is needed.  It should be enough to have `asyncio_tls_open_connection_readline.py`.